test/functional/test_framework/messages.py

```diff
@@ -1294,8 +1294,11 @@ class msg_tx:
     __slots__ = ("tx",)
     msgtype = b"tx"
 
-    def __init__(self, tx=CTransaction()):
-        self.tx = tx
+    def __init__(self, tx=None):
+        if tx is None:
+            self.tx = CTransaction()
+        else:
+            self.tx = tx
```

Comment:
Maybe a comment is warranted so that in the future somebody does not "simplify" it by reverting this patch. Maybe not necessary if https://github.com/bitcoin/bitcoin/pull/30553 enforces it.

Reply:
Right, https://github.com/bitcoin/bitcoin/pull/30553 is much better by a comment :)